modules/ROOT/pages/configure-client-management-openid-task.adoc

```diff
@@ -32,7 +32,7 @@ The *Add OIDC client provider* page appears.
 * *Dynamic Client Registration*
  ** *Issuer*: URL that the OpenID provider asserts is its trusted issuer.
  ** *Client Registration URL*: The URL to dynamically register client applications as a client application for your identity provider.
- ** (Optional) Click *Advanced settings* to add an *Authorization Header* for dynamic client registration requests. This header is required if the provider restricts registration requests to authorized clients. +
+  ** *Authorization Header*: The header to provide credentials to authenticate with a server. This header is required if the provider restricts registration requests to authorized clients. +
   *** For Okta, this value is `SSWS ${api_token}`, where `api_token` is an API token created through Okta.
   *** For ForgeRock, this value is `Bearer ${api_token}`, where `api_token` is an API token created through ForgeRock.
   *** For Salesforce, this value is `Bearer ${api_token}`, where `api_token` is an API token created through Salesforce.
```

Comment:
We can add one item for Azure, For Azure AD, this value is Bearer ${api_token}, where api_token is an API token created through Azure.